### ISSUE TITLE
Portraits of entities

### ISSUE THREAD

**Issue by dpitti** [snac-cooperative/snac#85]
Currently the images associated with an entity are derived via Wikidata. Thus only those entities with matches in Wikidata have a portrait (or associated image). While we do plan to one day be able to contribute identities to Wikipedia/Wikidata, and perhaps contribute portraits, we also need to consider being able to associate portraits within SNAC. What technically would we need to do to enable this? What semantics would we use on an archival resource link to a portrait? That is, what will we call the type, and the arc role? I am inclined to think that the role would be

xlink:role="http://socialarchive.iath.virginia.edu/control/term#DigitalArchivalResource”

Would the portrait ever be considered other than a archival resource? Even a photograph of a work of art would be an archival record, as it is not the artwork, but a record of the artwork. 

And for the arc role, I would suggest:

xlink:arcrole="http://socialarchive.iath.virginia.edu/control/term#digitalImageOf"

Or some such. What does wikidata call it? FOAF calls it foaf:depiction

http://xmlns.com/foaf/spec/#term_depiction

Ultimately I would like to use a RiC-O property for this, but the issue has not yet been discussed by EGAD. I will raise the issue with the RiC-O group.

**Comment by deternitydx** [comment]
Right now, our data structure does have an Image object that is passed along in a Constellation.  That is, a Constellation has a set of Images.  The Wikimedia Commons image links aren't stored in the Postgres database, but they are cached in the Elastic Search index to allow for the explore page.  Those links there are passed through to the webui in the Constellation's Image object.

My inclination would be to upgrade this Image object and a data store to allow us to keep images local (with their originating URL).  We would need to be concerned about copyright if users can upload images they find online.  When we export the Constellation to EAC-CPF, we could export the image/portrait as a resourceRelation.

I would hesitate to store them with the resources because then we will need to loop through all resources of a constellation to display the image/portrait.  For identities with a large number of resources (i.e. Washington, Henry, ...), we already must delay the loading of resource relations to speed up the page load time.

**Comment by deternitydx** [comment]
see https://github.com/snac-cooperative/snac/blob/master/src/snac/data/Image.php for Image.  Right now, it includes what is needed for Wikidata's policies on citing images and authors.

**Comment by dpitti** [comment]
Thanks much. I had intended to mention policy data, but forgot by the time I finished. I think the policy will be the same as for the descriptive data: CC0. Thus any editor that attaches an image must state they have rights to make the image available in the public domain. We need to give size and image quality recommendations. It ought to be 72 dpi or maybe a little higher. That is, people can use the image but quality will limit that use. If any researcher wants a higher quality, then they would have to go to the source. Much to think about here, as the image ought to have some descriptive metadata and information on how to contact entity with the rights to the image to request higher quality. I think. Minimal essential data. ID it, rights, contact.

**Comment by dpitti** [comment]
https://github.com/snac-cooperative/snac/blob/master/src/snac/data/Image.php would seem to capture the essential.

**Comment by deternitydx** [comment]
This does open a can of worms.  We could also consider IIIF image servers, or that users must upload to one of those servers and provide us a link.

We don't need to be as restrictive as CC0.  We could follow the same protocols as Wikidata (since we are currently doing that), and allow CC0 and CC-BY (where we link to the author and copyright notice).

**Comment by dpitti** [comment]
Good points. Though I understand how you are using "as restrictive as CC0," given that it is without restriction, it amused me. Yes, we could go with CC0 or CC-By, though we would need to revisit the overall policy statement, which is CC0. CC0 is problematic for some members, including ANF, as the bioghist and other prose/discursive items are considered copyrightable, even most elements of the description are not. We may change to CC-By at some point, at least for some elements of the description.

**Comment by deternitydx** [comment]
We do need to be careful.  There's a difference, I presume, between using images that have restrictions and storing/providing images that have restrictions.  If we just link to them in other, permanent, locations, that may be preferable.  (However, we already have some Wikimedia Commons image URLs that we've cached in the Elastic Search index removed from Wikimedia).

**Comment by dpitti** [comment]
Yes, this ought to be discussed with members. I incline to using our own IIIF server, with contributor filling in a brief form with the essentials, and one of the essentials would be a URL if the image is also locally served within an archival context. But we have a copy locally, to ensure reliability.
